ios/RCTImageResizer/RCTImageResizer.m

```diff
@@ -40,11 +40,17 @@ void saveImage(NSString * fullPath, UIImage * image, float quality)
                   callback:(RCTResponseSenderBlock)callback)
 {
     CGSize newSize = CGSizeMake(width, height);
-    NSString* fullPath = generateCacheFilePath(@".jpg");
+    NSString* fullPath = generateCacheFilePath(@"jpg");
 
     [_bridge.imageLoader loadImageWithTag:path callback:^(NSError *error, UIImage *image) {
         if (error || image == nil) {
-            image = [[UIImage alloc] initWithContentsOfFile:path];
+            UIImage* image;
```

Comment:
i think you should delete this line, otherwise when doing the resizing UIImage * scaledImage = [image scaleToSize:newSize];, it will be on a null image